gradient/cli/notebooks.py

```diff
@@ -24,72 +24,83 @@ def notebook_metrics():
     pass
 
 
-# @notebooks_group.command("create", help="Create new notebook")
-# @click.option(
-#     "--vmTypeId",
-#     "vm_type_id",
-#     type=int,
-#     required=True,
-#     help="Type of Virtual Machine",
-#     cls=common.GradientOption,
-# )
+@notebooks_group.command("create", help="Create new notebook")
 @click.option(
-    "--containerId",
-    "container_id",
-    type=int,
+    "--clusterId",
+    "cluster_id",
+    type=str,
     required=True,
-    help="Container ID",
+    help="Cluster ID",
     cls=common.GradientOption,
 )
 @click.option(
-    "--clusterId",
-    "cluster_id",
+    "--vmTypeLabel",
+    "vm_type_label",
+    type=str,
+    help="Virtual Machine type label e.g. P5000",
+    cls=common.GradientOption,
+)
+@click.option(
+    "--vmTypeId",
+    "vm_type_id",
     type=int,
-    required=True,
-    help="Cluster ID",
+    help="Virtual Machine type id",
+    cls=common.GradientOption,
+)
+@click.option(
+    "--containerId",
```

Comment:
This is similar to my comment above

Reply:
This one is weird to get for the user but we do want to support because its the only way to user the containers that show up on the "create notebook" page. We expect most users to use `containerName`.